### ISSUE TITLE
Will the topology of the network changed dynamically in the test?

### ISSUE THREAD

**Issue by WanderingTroubadour** [SUSTech-HPCLab/CS305-2023Spring-Project#5]
I.e. are there addition/modification/deletion of links/hosts/switches after network has been built?

Or, whether we are required to implement the dynamical modification of the flow table?

**Comment by EricLee543** [comment]
Yes. You need to consider situations where the network topology changes dynamically. The project also asks you to provide a complex testcase  to demonstrate the robustness of your program. You can consider this dynamically changing situation

**Comment by WanderingTroubadour** [comment]
> 

Then whether we need to consider the complexity of these operations?

**Comment by EricLee543** [comment]
The data of the test cases are not very large. No need to consider time complexity
